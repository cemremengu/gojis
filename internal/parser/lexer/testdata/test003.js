console.log("some string parameter");
console.log("some string parameter", "with another one");
"just a string"
"this one has some \0x99 escaped characters in it"