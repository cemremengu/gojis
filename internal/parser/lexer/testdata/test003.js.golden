IdentifierName
Dot
IdentifierName
ParOpen
StringLiteral
ParClose
SemiColon
LineTerminator
IdentifierName
Dot
IdentifierName
ParOpen
StringLiteral
Comma
Whitespace
StringLiteral
ParClose
SemiColon
LineTerminator
StringLiteral
LineTerminator
StringLiteral
